molgenis-app/src/main/java/org/molgenis/app/promise/mapper/ParelMapper.java

```diff
@@ -280,14 +290,29 @@ private Entity toAgeType(String ageType)
 		List<String> unknown = Lists.newArrayList();
 		for (Entity sample : promiseSampleEntities)
 		{
-			String type = sample.getString("MATERIAL_TYPES");
-			if (materialTypesMap.containsKey(type))
+			String tissue = sample.getString("MATERIAL_TYPES_SUB");
```

Comment:
```MATERIAL_TYPES_SUB``` will be "-1"  when there is no tissue stored. We only look up the tissue type when it is not "-1".